webapp/backend/views.py

```diff
@@ -955,7 +955,18 @@ def destroy(self, request, uuid, format=None):
                                              .format(state="destroyed"))
 
         if lambda_instance_data['status'] == LambdaInstance.CLUSTER_FAILED:
-            raise CustomCantDoError(CustomCantDoError.messages['cant_do'].
+            if 'out of limit' in lambda_instance_data['failure_message']:
+                events.set_lambda_instance_status.delay(lambda_instance_data['uuid'],
```

Comment:
Considering that the web UI doesn't show destroyed lambda instance, wouldn't this change just make the lambda instance disappear? It would be `building` and suddenly would just disappear.